core/neat/mixins/_grid-container.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+
+/// Creates Neat grid container with clearfix.
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-container;
+///   }
+///
+/// @example css
+///   .element {
+///     margin-left: auto;
+///     margin-right: auto;
+///   }
+///
+///   .element::after {
+///     clear: both;
+///     content: "";
+///     display: block;
+///   }
+
+@mixin grid-container($grid: $neat-default-grid) {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
@wardpenney Yeah. To clarify, I think it's totally fine for this mixin to just be a clearfix. It makes for a good API for the grid system and prevents us from having to say “Oh, don't forget when you use grid columns, that you need to use a clearfix!”.

:+1: